AwsCryptographyPrimitives/src/HKDF/HMAC.dfy

```diff
@@ -19,38 +19,38 @@ module {:options "-functionSyntax:4"} {:extern "HMAC"} HMAC {
     ghost function {:extern} GetDigest(): Types.DigestAlgorithm reads this
     ghost function {:extern} GetInputSoFar(): seq<uint8> reads this
 
-    // A build method is used insted of a constructor
+    // A build method is used instead of a constructor
     // because in Java creating a `Mac` object
     // can throw because the Java function takes a string.
     // Dafny constructors MUST succeed so this mismatch
     // make a static Build method required.
-    static method {:extern} Build(digest: Types.DigestAlgorithm)
+    static method {:extern} {:axiom} Build(digest: Types.DigestAlgorithm)
```

Comment:
Q: Is there any performance improvement when using axiom?

Reply:
Dafny emits a warning if you have an `extern` with an `ensures` and you need an axiom to silence the warning.